components/postmark/postmark.app.mjs

```diff
@@ -1,11 +1,208 @@
+import axios from "axios";
```

Comment:
I initially had problems to pass the $ parameter since it seemed to be available only in the `run` method, but I followed the provided examples and eliminated the "duplicate" axios module as recommended.

Reply:
Sure, we generally use `this` as fallback if `$` is not available